api/reference/provision_requests.adoc

```diff
@@ -80,6 +82,81 @@ Example: Store a request_id from an external system so the system can be notifie
 | miq_custom_attributes | hash | Custom attributes applied to the virtual machine and stored in the database as part of provisioning
 |=====================
 
+[[editing-requests]]
+=== Editing Requests
+
+Updating provision requests can be performed by posting an _edit_ action as follows:
+
+[source,data]
+----
+POST /api/provision_requests/11
+----
+
+[source,json]
+----
+{
+  "action" : "edit",
+  "resource" : {
+    "options" : {
+      "custom_var1" : "custom value1"
+    }
+  }
+}
+----
+
+==== Response:
+
+[source,json]
+----
+{
+  "href": "http://localhost:3000/api/provision_requests/11",
+  "id": "3068",
```

Comment:
just for consistency, this should be "11" I believe?

Reply:
👍